src/main/resources/templates/apidocumentation/apiDocumentationStartPage.ftl

```diff
@@ -131,6 +131,16 @@
         to see what's changed and update your application if necessary.
       </p>
 
+      <h2 class="govuk-heading-m" id="cors">
+        Cross-origin requests
+      </h2>
+      <p class="govuk-body">
+        Cross-origin requests are not supported by the API, so you can't make API requests directly from client side code in a web browser.
```

Comment:
Could give an explanation here as well e.g. 'This is to ensure the IP address based rate limiting is as effective as possible.' if you think thats necessary? Not sure on the wording.

Reply:
I don't think we need an explaination but I think it's worth mentioning JavaScript in this paragraph so it shows up in a ctrl+f for 'javascript' if they wonder why it's not working:

`..from client-side JavaScript code in a web browser.`